HostSupport/src/ofxhImageEffectAPI.cpp

```diff
@@ -282,10 +282,11 @@ namespace OFX {
 #           ifdef OFX_DEBUG_ACTIONS
               OfxPlugin *op = _pluginHandle->getOfxPlugin();
               std::cout << "OFX: "<<op->pluginIdentifier<<"("<<(void*)op<<")->"<<kOfxActionUnload<<"()"<<std::endl;
+              std::string pluginIdentifier(op->pluginIdentifier); // save it before it becomes invalid
 #           endif
             stat = (*_pluginHandle)->mainEntry(kOfxActionUnload, 0, 0, 0);
 #           ifdef OFX_DEBUG_ACTIONS
-              std::cout << "OFX: "<<op->pluginIdentifier<<"("<<(void*)op<<")->"<<kOfxActionUnload<<"()->"<<StatStr(stat)<<std::endl;
```

Comment:
Thanks for fixing that!

Reply:
It's it still a draft or is it ready to merge?

Reply:
It is still a draft as I am not sure of the way `StrChoiceInstance::notify` is supposed to work.

Reply:
About `openfx-misc` plugins, what is the issue ? They do not load on macOS and Windows and Linux or a specific OS and specific versions of Resolve ?